packages/app/src/cli/models/app/app.test-data.ts

```diff
@@ -265,7 +266,17 @@ export async function testPaymentExtensions(directory = './my-extension'): Promi
   return extension
 }
 
-export async function testWebhookExtensions(emptyConfig = false): Promise<ExtensionInstance> {
+export function testWebhookExtensions(params?: {
+  emptyConfig?: boolean
+  complianceTopics: false
+}): Promise<ExtensionInstance>
+export function testWebhookExtensions(params?: {
+  emptyConfig?: boolean
+  complianceTopics: true
+}): Promise<ExtensionInstance[]>
+export async function testWebhookExtensions({emptyConfig = false, complianceTopics = false} = {}): Promise<
```

Comment:
I agree on most occasions, this is unfortunately only like this because the two modules share configuration and sometimes depend on eachother. The flags passed in to control test data (these functions are at most only generating dummy test data) is also a pattern we tend use, usually when mocking out GQL requests. I utilizied an overload to preserve the other tests using this helper.